docs/command-reference/strings/append.md

```diff
@@ -10,34 +10,33 @@ import PageTitle from '@site/src/components/PageTitle';
 
 ## Introduction
 
-The `APPEND` command in Redis/Dragonfly is used to add a specified value to the end of a string stored at a given key. If the key doesn't exist, it creates it and initializes it with an empty string before appending. This command is particularly useful for accumulating logs or constructing strings incrementally.
+In Dragonfly, as well as in Redis and Valkey, the `APPEND` command is used to add a specified value to the end of a string stored at a given key.
```

Comment:
We can use multiple lines when it gets long.
I personally prefer one sentence per line in markdown, and the rendered version looks the same.